src/trufont/windows/fontWindow.py

```diff
@@ -600,6 +597,8 @@ def OnSaveAs(self, event):
             if dialog.ShowModal() == wx.ID_CANCEL:
                 return
             path = dialog.GetPath()
+            if not path.endswith(".tfont"):
+                path += ".tfont"
```

Comment:
This is unfortunate, because say if you're saving to foo the dialog has not checked that foo.tfont doesn't already exist, if which case it'll be overwritten silently. I don't think I've needed this line on windows, but if it's needed elsewise...

Reply:
Hrm. Maybe I need to ask around about this...